common/credCache_darwin.go

```diff
@@ -25,7 +25,7 @@ import (
 	"fmt"
 	"sync"
 
-	"github.com/wastore/keychain" // forked and customized from github.com/keybase/go-keychain, todo: make a release to ensure stability
+	"github.com/keybase/go-keychain" // forked and customized from github.com/keybase/go-keychain, todo: make a release to ensure stability
```

Comment:
yes, Az identity-cache module  brings github.com/keybase/go-keychain dependency with itself , we had removed github.com/wastore/keychain dependency because it was causing build failures and co-existing issues , hence we had to fall back to this dependency in order to avoid conflicts.

Reply:
it says wastore is forked and customized from keybase, what are the customizations that were made and do they need to exist to continue to work with storage explorer?

Reply:
github.com/wastore/keychain- is a slim library for accessing the Keychain for macOS in Go its build ontop of github.com/keybase/go-keychain it contains wrappers around the functionalities present in wastore/keychain . I dont believe it should cause a problem

Reply:
Does this code compile and work on Mac ?

Reply:
Yes, we have made test runs over a macOS-vm to verify the correctness and stability.
Test runs are attached over in the Design Doc for reference.